manual/order.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+require_once ('../razorpay-php/Razorpay.php');
+use Razorpay\Api\Api;
+
+session_start();
+
+$keyId = '<Enter your key id>';
+$keySecret = '<Enter your key secret>';
+
+$api = new Api($keyId, $keySecret);
+
+$orderData = [
+    'receipt'         => 3456,
+    'amount'          => 2000 * 100, // 2000 rupees in paise
+    'currency'        => 'INR',
+    'payment_capture' => 1 // auto capture 
+];
+
+$razorpayOrder = $api->order->create($orderData);
+
+$razorpayOrderId = $razorpayOrder['id'];
+
+$_SESSION['razorpay_order_id'] = $razorpayOrderId;
+
+$amount = $orderData['amount'];
+
+$html = <<<EOT
+<button id="rzp-button1">Pay with Razorpay</button>
+<script src="https://checkout.razorpay.com/v1/checkout.js"></script>
+<form name='razorpayform' action="verify-signature.php" method="POST">
+    <input type="hidden" name="razorpay_payment_id" id="razorpay_payment_id">
+    <input type="hidden" name="razorpay_signature"  id="razorpay_signature" >
+</form>
+<script>
+var options = {
```

Comment:
We want to support both automatic and manual checkout but only with orders API. The automatic/index.html is still on old integration, and the new one is only on manual checkout.